client/src/js/services/FinancialService.js

```diff
@@ -0,0 +1,39 @@
+angular.module('bhima.services')
+.service('FinancialService', FinancialService);
+
+FinancialService.$inject = [ '$http', 'util' ];
+
+function FinancialService ($http, util) {
+  var service = this;
+
+  service.readCostCenter = readCostCenter;
```

Comment:
What is the difference between `readCostCenter()` and `getCostCenter()`?  They are name very similarly.

Could you include comments in the code describing the difference?